### PR TITLE
added the --all flag on `ord wallet sats`

### PR DESCRIPTION
**Pull request by cryptoni9n**
further addressing #3792 

return all sat ranges in wallet, in addition to
the rare sats the command normally returns.

### CODE REVIEW

File: src/subcommand/wallet/sats.rs [hunk 1]
```diff
@@ -32,6 +38,25 @@ impl Sats {
...
+        .iter()
+        .flat_map(|(_, ranges)| ranges)
+        .cloned()
+        .collect();
+
+      ranges.sort_by(|a, b| a.0.cmp(&b.0));
+
+      let formatted_ranges: Vec<String> = ranges
+        .iter()
+        .map(|range| format!("{} - {}", range.0, range.1))
```
Comment (raphjaph) [inline src/subcommand/wallet/sats.rs:52]:
```suggestion
        .map(|range| format!("{}-{}", range.0, range.1))
```

File: src/subcommand/wallet/sats.rs [hunk 2]
```diff
@@ -7,6 +8,11 @@ pub(crate) struct Sats {
     help = "Find satoshis listed in first column of tab-separated value file <TSV>."
   )]
   tsv: Option<PathBuf>,
+  #[arg(
+    long,
+    help = "Display list of all sat ranges in wallet and special sats."
```
Comment (raphjaph) [inline src/subcommand/wallet/sats.rs:13]:
```suggestion
    help = "Display list of all sat ranges in wallet"
```

**Review by raphjaph — CHANGES_REQUESTED** [review]
Looks good, just a couple nits and also still needs a test

File: src/subcommand/wallet/sats.rs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 use super::*;
+use clap::Parser;
```
[the remark this answers is not in the document]
Comment (cryptoni9n) [reply src/subcommand/wallet/sats.rs]: updated to remove redundant use statement

**Review by raphjaph — APPROVED** [review]
LGTM

**Pull request merged**